src/NuGet.Core/NuGet.Packaging/PackagingConstants.cs

```diff
@@ -24,6 +24,7 @@ public static class Folders
             public static readonly string ContentFiles = "contentFiles";
             public static readonly string Lib = "lib";
             public static readonly string Native = "native";
+            public static readonly string Any = "any";
```

Comment:
This string doesn't appear to be used any more. If that's correct, it should be removed.